dev/MenuBar/MenuBar_InteractionTests/MenuBarTests.cs

```diff
@@ -375,6 +375,30 @@ public void TabTest()
             }
         }
 
+        [TestMethod]
+        public void EmptyMenuBarItemNoPopupTest()
+        {
+            if (PlatformConfiguration.IsDevice(DeviceType.Phone))
```

Comment:
Do we still need that? I don't think we are doing any tests on Windows Mobile devices any longer.

Reply:
I thought the same. I think we should look at all test holistically at some point and remove obsolete checks.

Reply:
This does look like dead code. @kmahone to verify this codepath is not hit in any of the configurations we run tests in.